manager/src/test/resources/application.properties

```diff
@@ -17,7 +17,4 @@ quarkus.flyway.connect-retries=10
 quarkus.flyway.migrate-at-start=true
 quarkus.flyway.validate-on-migrate=true
 quarkus.flyway.ignore-missing-migrations=true
-quarkus.flyway.create-schemas=true
-quarkus.hibernate-orm.log.sql=true
```

Comment:
@danielezonca Could you keep these for the `dev` profile. It's useful to see the SQL hibernate is generating when developing against the DB.

Reply:
Done 👍